Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM calico/build:latest
+MAINTAINER projectcalico
+
+ADD . /code
+WORKDIR /code
+
+ARG K8S_VERSION=local
+ENV K8S_VERSION ${K8S_VERSION}
+
+RUN pip install pykube && pyinstaller calico.py -ayF && \
```

Comment:
I think that rather than building the binaries within this container, I'd prefer using the binaries already built and placed in the `dist` directory.

So, something like this instead:
```
ADD ./dist/calico /opt/cni/bin
ADD ./dist/calico-ipam /opt/cni/bin
```